### PR TITLE
FT Assignments API app

### PR DESCRIPTION
**Pull request by mugiwaramunyi**
**What does this PR do?**

This adds an Assignments API app to the Django project CitizenReporter_API. This API is composed of two endpoints:
-/assignments
-/assignments/<id>

**Description of Task to be completed?**

Create an API endpoint to create, update, delete and return a list of current assignments. This endpoint should be publicly available to anyone

**How should this be manually tested?**

`python manage.py test assignments.tests`

Any background context you want to provide?

This project is being re-engineered from scratch, therefore all code added to this project is new.

### CODE REVIEW

**Comment by phillipahereza** [comment]
I see in your tests, when creating an assignment the post data includes the `id`, the ID is supposed to be automatically generated by the django ORM when it creates an entry in the database for that assignment

**Comment by mugiwaramunyi** [comment]
Noted. I also thought about this when writing my code, but I forgot to seek
further clarification. Will refactor tonight.

**Comment by phillipahereza** [comment]
@Jonathan-Ngondi could you also test the assignments featured image upload. Mock it 
https://swapps.io/blog/testing-files-with-pythondjango/

**Comment by mugiwaramunyi** [comment]
Okay will try it out.

File: assignments/views.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from assignments.models import Assignment
```
Comment (phillipahereza) [inline assignments/views.py]: run `isort -rc --atomic -y .` to sort your imports as specified in the `PEP8` standard

**Pull request closed without merging**